LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
what should these values be?

Reply:
it's can also be nodejs project ?

Reply:
OpenJS own the copyright for node and its related projects, so I think it needs to be whatever their thing is.

Reply:
Got the answer: https://github.com/openjs-foundation/cross-project-council/blob/HEAD/governance/IP_POLICY_GUIDANCE.md#2-copyright-notices